src/meshes/p4est_cubed_sphere.jl

```diff
@@ -20,16 +21,21 @@ The mesh will have no boundaries.
                       The mapping will be approximated by an interpolation polynomial
                       of the specified degree for each tree.
 - `RealT::Type`: the type that should be used for coordinates.
-- `initial_refinement_level::Integer`: refine the mesh uniformly to this level before the simulation starts.
+- `initial_refinement_level::Integer`: refine the mesh uniformly to this level before the
+  simulation starts.
 - `unsaved_changes::Bool`: if set to `true`, the mesh will be saved to a mesh file.
-- `p4est_partition_allow_for_coarsening::Bool`: Must be `true` when using AMR to make mesh adaptivity
-                                                independent of domain partitioning. Should be `false` for static meshes
-                                                to permit more fine-grained partitioning.
+- `p4est_partition_allow_for_coarsening::Bool`: Must be `true` when using AMR to make mesh
+  adaptivity independent of domain partitioning. Should be `false` for static meshes to
+  permit more fine-grained partitioning.
+- `element_local_mapping::Bool`: option to use the element-local mapping from Appendix A of
+  [Guba et al. (2014)](https://doi.org/10.5194/gmd-7-2803-2014).
```

Comment:
Can you please add a comment here stating what this mapping does (bi-linear mapping + scaling) and what the use of this particular mapping is?

For now, I think that it would be sufficient to mention that this is an alternative approach to constructing the mapping. However, a more detailed explanation of why this type of mapping is necessary should be added before merging #31.

Reply:
I have added a more detailed description of the mapping in [a7a6558](https://github.com/trixi-framework/TrixiAtmo.jl/pull/34/commits/a7a65583c784cbf2a5f2ac2d5bb2f4067ddd55b9) and [ece11a8](https://github.com/trixi-framework/TrixiAtmo.jl/pull/34/commits/ece11a8c7bfdc8e13301bd35249e9669f9b18146).